a11y-input/src/ally-input.js

```diff
@@ -13,6 +13,22 @@ export class A11yInput extends LitElement {
     this.label = '';
   }
 
+  /**
+   * Anti-Pattern for testing the `Light DOM`,
+   * though useful for testing a11y 🤷‍♂️😳
+   */
+  connectedCallback () {
+    super.connectedCallback();
+    this.labelEl = document.createElement('label');
+    this.labelEl.innerText = this.label;
+    this.labelEl.setAttribute('slot', 'label');
+    this.appendChild(this.labelEl);
+
+    this.inputEl = document.createElement('input');
+    this.inputEl.setAttribute('slot', 'input');
+    this.appendChild(this.inputEl);
+  }
```

Comment:
### Added `connectedCallback()` 

From article:
> Note: Using the `DOM API` in a lit-element's lifecycle (here in`connectedCallback`) is an **anti-pattern**, however to allow for _a11y_ it might be a real use case.